tests/installation/yast_fullscreen.pm

```diff
@@ -0,0 +1,20 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Enlarge the YaST window for fullscreen in ssh-X test.
```

Comment:
let's name it `ensure_installer_fullscreen` wdyt? similar than the method name, this functionality is not very YaSTish' IMO :) . In the future someone could pass specific string to search for screens. Also it is not clear which yast screen in particular as there are popups and more windows.